docs/src/man/joins1.md

```diff
@@ -0,0 +1,609 @@
+
+`Joins` is another very common and important operation that arises in the world of tabulated data. 
+A join across two DataFrames is the action of combining the two datasets based on shared column values 
+that exist across the two tables. We call this column (or columns) the `key`. So, each record from the 
+first table is matched to a record in the second table — as long as the records’ values are the same. 
+
+I’m sure you’re quite familiar with e-commerce sites like Amazon and Flipkart these days. We are 
+bombarded by their advertisements when we’re visiting non-related websites – that’s the power of 
+targeted marketing! Previously, we have used German Credit Card dataset. Now let's say you are working
+on a Classification problem but we have not target column in our dataset. We are given two data frames – 
+one which contains data about customer and the other that has taget information. We will use these dataframes 
+to understand how the different types of joins work using `DataFrames.jl`.
+
+For example, suppose that we have the following two datasets:
+
+```jldoctest dataframe
+julia> using DataFrames
+
+julia> using CSV
+
+julia> german_ref = CSV.read((joinpath(dirname(pathof(DataFrames)),
+                                    "..", "docs", "src", "assets", "german.csv")),
+                                    DataFrame)
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67  male        2  own      NA               little          ⋯
+    2 │     1     22  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+```
+
+This table has all the `ids` of all factors to make comparisions for risk factor. Let’s say we 
```

Comment:
it is not clear what you mean by "factor". Also why do you earlier discussed some marketing application so why do we talk about risk here? Also why Risk is capitalized?